api/DOMQuad.json

```diff
@@ -29,10 +29,10 @@
             "version_added": "45"
           },
           "safari": {
-            "version_added": true
+            "version_added": "11"
           },
           "safari_ios": {
-            "version_added": true
+            "version_added": "11"
```

Comment:
Based upon the linked changeset and its presence in WebKit 603.1.30, shouldn't this be Safari for iOS 10.3 (WebKit 603.2.1)?  (This applies to the entire file.)